### ISSUE TITLE
Pokemon models in HD

### ISSUE THREAD

**Issue by donbailon** [Superviral/Pokemon-GO-App-Assets-and-Images#1]
Is it possible to have de Pokemon models in big resolution?

Good jobs guys! It's amazing. 🙌

**Comment by Superviral** [comment]
I'm sure there are some super HD renders floating around the Internet, but if I come across any, I'll add them to the repo!

**Comment by donbailon** [comment]
Yep! I think the same, but for example if you need to include this assets in an app, you need more resolution... I will continue looking for a HD assets... 

Thanks for all dude!

**Comment by iounpaladin** [comment]
Are there .obj-s for the Pokemon? There seem to only be .png-s.
